plugin-flex-ts-template-v2/src/feature-library/custom-transfer-directory/config.ts

```diff
@@ -0,0 +1,55 @@
+import { getFeatureFlags } from '../../utils/configuration';
+
+const {
+  enabled = false,
+  use_legacy_search_icon = false,
+  queue: {
+    enabled: queueEnabled = false,
+    show_only_queues_with_available_workers = false,
+    show_real_time_data = false,
+    enforce_queue_filter_from_worker_object = false,
+    enforce_global_exclude_filter = false,
+    global_exclude_filter = '',
+  },
+  worker: { enabled: workerEnabled = false },
+} = getFeatureFlags()?.features?.custom_transfer_directory || {};
+
+export const isFeatureEnabled = (): boolean => {
+  return enabled;
+};
+
+export const isCustomQueueTransferEnabled = (): boolean => {
+  return isFeatureEnabled() && queueEnabled;
+};
+
+export const isCustomWorkerrTransferEnabled = (): boolean => {
```

Comment:
Keyboard thing happened! But, this should be removed until the worker directory is implemented anyway.